src/ethereum_test_specs/blockchain.py

```diff
@@ -440,7 +441,9 @@ def generate_block_data(
         )
 
         try:
-            rejected_txs = verify_transactions(txs, transition_tool_output.result)
+            rejected_txs = verify_transactions(
+                t8n.exception_mapper, txs, transition_tool_output.result
+            )
```

Comment:
I think we'll also need to add `verify_transactions()` to `src/ethereum_test_specs/state.py`.

Reply:
yes this is so far the blockchain version of tests only

Reply:
fixed